### DISCUSSION TITLE
Does anyone offer consulting?

### DISCUSSION THREAD

**Discussion by c00lcoder** [saleor/saleor#12299] [Q&A]
I am exploring using saleor for a digital download e-commerce store. Does anyone offer consulting services that could point me in the right direction and/or get me going with a good code base to setup a digital download store? Digital downloads are images, videos, audio, etc...

**Comment by timuric** [comment]
Hi @c00lcoder you could check partner agencies on this page https://saleor.io/experts/
Also feel free to reach out us by email hello@saleor.io

**Comment by timuric** [comment]
You can check recently launched digital store https://www.colophon-foundry.org/ developed by https://madebyon.com/

**Comment by michalmirumee** [comment]
Hi @c00lcoder! We could definitely help. Would you like to have a quick chat?

**Reply by c00lcoder** [reply]
Hi, yes I'd be open to a chat

**Reply by michalmirumee** [reply]
Cool @c00lcoder ! Write me a DM: michal.sukiennik@mirumee.com
